lib/hanami/cli/generators/app/action.rb

```diff
@@ -56,6 +56,14 @@ def call(app, controller, action, url, http, format, skip_view, slice, context:
           }.freeze
           private_constant :ROUTE_RESTFUL_URL_SUFFIXES
 
+          # @api private
+          # @since 2.1.0
+          RESTFUL_ACTIONS = {
```

Comment:
@timriley I renamed via commit, and I am not accepting your PR suggestion because I had to rename the constant references in the file.

See https://github.com/hanami/cli/pull/114/commits/52ca7c455da0a850be606536aabd016e82e5fb6e